src/main/java/io/confluent/connect/hdfs/parquet/ParquetHiveUtil.java

```diff
@@ -79,6 +82,7 @@ private Table constructParquetTable(
     }
     // convert Connect schema schema to Hive columns
     List<FieldSchema> columns = HiveSchemaConverter.convertSchema(schema);
```

Comment:
we dont need to filter the schema here?

Reply:
Not in this case, as the `schema` is not used later like in the `AvroHiveUtil` for `table.getParameters()`. Now that you mention it though, it looks like we could simply filter the schema and then we don't need the `removeFieldPartitionColumn` to call on the columns.